components/app/Slack.vue

```diff
@@ -171,6 +171,7 @@ const selected = ref(channels[0])
         <div class="absolute bottom-4">
           <AppButton
             :to="SLACK_URL"
+            :target="'_blank'"
```

Comment:
we can just use it like:

```suggestion
            target="_blank"
```